tldw_Server_API/tests/Character_Chat_NEW/integration/test_character_chat_auto_routing.py

```
@@ -0,0 +1,99 @@
+
+    chat_resp = test_client.post(
+        "/api/v1/chats/",
+        json={"character_id": char_id, "title": "Auto route integration"},
+        headers=auth_headers,
+    )
+    assert chat_resp.status_code == 201
+    chat_id = chat_resp.json()["id"]
+
+    captured: dict[str, object] = {}
+
+    def _stub_chat_api_call(api_endpoint, messages_payload, **kwargs):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_stub_chat_api_call`

(ANN202)